### ISSUE TITLE
[webhooks] Wrong example payload for `security_and_analysis` event

### ISSUE THREAD

**Issue by wolfy1339** [github/docs#18668]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#security_and_analysis

### What part(s) of the article would you like to see updated?

The payload example for the GitHub.com version of the docs has a payload for GHES.

Normally payloads for GitHub.com don't have an `enterprise` key, and there are other keys in the `repository` key in the payload that aren't present in the other example payloads.

Specifically the following are the keys in the `repository` property that need clarification:
- `repository[network_count]` is new and isn't seen on other payloads for GitHub.com webhooks
- `repository[subscribers_count]` is new and isn't seen on other payloads for GitHub.com webhooks
- `repository[organization]` is an `object` while in other payloads it is a `string`

### Additional information

I discovered this discrepancy while creating a schema for this new event over in https://github.com/octokit/webhooks/pull/641

**Comment by timrogers** [comment]
@wolfy1339 Thanks for taking the time to report this! I'm the Product Manager for Webhooks at GitHub. I'll pick this up with my team and will get this fixed as soon as we can.

**Comment by timrogers** [comment]
@wolfy1339 Have you been able to test this webhook, or are you just looking at the example? I'm wondering if the example is actually wrong or if it is just different to others (which might be a bad thing or a mistake!).

**Comment by wolfy1339** [comment]
I haven't been able to trigger the webhook myself to test it properly.

> I'm wondering if the example is actually wrong or if it is just different to others (which might be a bad thing or a mistake!).

It's different than others, not completely wrong. It's not in the expected format either, the payload has an extra `event` key, and the actual payload is wrapped inside the `payload` key, instead of being at the root of the JSON

![image](https://user-images.githubusercontent.com/4595477/174153051-125ce85f-e2ab-4511-b452-48fa1cc8a38a.png)


> Normally payloads for GitHub.com don't have an `enterprise` key, and there are other keys in the `repository` key in the payload that aren't present in the other example payloads.

This is what lead me to this conclusion ^.

> Specifically the following are the keys in the repository property that need clarification:
> - `repository[network_count]` is new and isn't seen on other payloads for GitHub.com webhooks
> - `repository[subscribers_count]` is new and isn't seen on other payloads for GitHub.com webhooks
> - `repository[organization]` is an object while in other payloads it is a string

Those properties are not present on any recent delivery of webhooks for GitHub.com

**Comment by timrogers** [comment]
Thanks for clarifying! That's super helpful - sounds like you just think this webhook is unusual so you wanted to confirm. Let me check up on this internally and get back to you.

**Comment by timrogers** [comment]
@wolfy1339 Watch this space - we're working on this issue.

**Comment by skedwards88** [comment]
Thanks for flagging this! I've opened an internal PR to remove the extra `event` key from the payload example. You can expect to see that change live sometime next week. We're checking with the team about the other keys in question.

**Comment by wolfy1339** [comment]
It seems the event was removed from the webhooks docs for GitHub.com (Free,Pro, Team)

Does this mean this event is only available to GHES users? If so, it would probably explain the other keys in question

**Comment by timrogers** [comment]
@wolfy1339 You're right - it has now disappeared! It is only available in GitHub Enterprise Cloud ([link](https://docs.github.com/en/enterprise-cloud@latest/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#security_and_analysis)), GHES and GitHub AE.
